classes/classes/BaseContent.as

```diff
@@ -43,7 +43,7 @@
 
 		protected function get output():Output
 		{
-			return kGAMECLASS.output;
+			return Output.init();
```

Comment:
That was kinda useless/unneccessary/stupid. Hence I reverted it. (See the next commit)